action/protocol/execution/evm/evm.go

```diff
@@ -395,7 +398,11 @@ func executeInEVM(ctx context.Context, evmParams *Params, stateDB *StateDBAdapte
 	if stateDB.Error() != nil {
 		log.L().Debug("statedb error", zap.Error(stateDB.Error()))
 	}
-	refund := (evmParams.gas - remainingGas) / 2
+	if !london {
```

Comment:
where is old code `(evmParams.gas - remainingGas) / 2`

Reply:
`params.RefundQuotient = 2`

Reply:
what if `params.RefundQuotient` is changed?

Reply:
check Ethereum implementation: https://github.com/ethereum/go-ethereum/pull/22733

Ethereum needs to keep the change backward-compatible (just like us), if it's changed past blocks won't be able to sync, why would they change?

Reply:
ok. Could you add the comment like theirs?

Reply:
done